src/borneo/operations.py

```diff
@@ -5132,6 +5354,35 @@ def add_exception(self, e):
         num = self.get_num_exceptions(e) + 1
         self._exception_map[e] = num
 
+    def get_exceptions_map(self):
+        # type: () -> dict[Object, int]
+        """
+        Internal use only.
+
+        Returns the map of exceptions.
+
+        :versionadded: 5.3.5
```

Comment:
update to next release (please find all of these)

Reply:
done